creatorkit/Toolbox.md

```diff
@@ -0,0 +1,157 @@
+
+Select the desired .hex file, and click Program to flash the board.
+
+## Creating a FlowM2M account
+
+In order to use your Creator Kit you will need to create a free account on [the FlowM2M website](http://www.flowcloud.io). This account allows you to manage your connected devices, and also interact with them remotely. You will use a device registration code from this site to provision devices using FlowM2M.
+
+## Provisioning Ci40
+
+Log in to your account on [the FlowM2M website](http://www.flowcloud.io) and view the Dashboard. From here you can navigate to the Devices page. This page will show all previously provisioned devices. In order to add a new device you need to click on the "Register" button, and then take a code.
+
+On your Ci40 you need to connect to a network via Ethernet to gain Internet access, and also access to the onboard web interface. Once connected to a network, you can type the following into the Ci40 terminal
+
+<code># ifconfig</code>
+
+The output of this command will include an eth0 "inet addr" which is the devices IP address.
+
+Using a different device connected to the same network you can now navigate to the Ci40 web interface by opening a browser and typing the IPaddress you got from the ifconfig command into the URL bar.
+
+You will be presented with a page that allows you to name the device, and also add a provisioning code. Use the code that you retrieved from the FlowM2M website earlier. Once you have provisioned you will be presented with a page allowing you to provision Clicker boards.
+
+## Provisioning Clicker
+
+Once your Ci40 is provisioned you can continue using the web interface to also provision Clicker boards if they have been programmed with Flowm2m enabled software. You will be presented with a list of 6LoWPAN devices that the Ci40 can find within range. If your Clickers are powered on, they should be displayed on this page. You can now provision these devices using FlowM2M. This step is required to make sure that only 6LoWPAN devices you want can access your datastores via Ci40.
+
+## Building Code from source
+
+Note: You do not need to do this to set up and run the example projects for Creator Kit. If you want to expand on the projects and eventually build your own it is recommended that you set up the first project and then come back to this section once you have seen the system in action. You can then move on to building your own IoT project using Creator Kit.
+
+Initially you need to install dependencies required to build. run the following the terminal on your Ubuntu build machine
+
+<code>$ sudo apt-get -y install g++ gcc binutils bzip2 flex python perl make libncurses5-dev libssl-dev grep unzip gawk subversion zlib1g-dev build-essential git mercurial cmake python-nose python-lxml</code>
+
+If you do not have repo installed run the following
+
+<code>$ curl https://storage.googleapis.com/git-repo-downloads/repo > ~/bin/repo</code>
+
+<code>$ chmod a+x ~/bin/repo</code>
+
+
```

Comment:
At the end of the document, it should point to https://github.com/CreatorKit/build/blob/dev/README.md for building the Creator Kit Project, rather than mentioning the steps for maintainability